src/api/controllers/guilds.ts

```diff
@@ -52,6 +52,14 @@ export async function handleInternalGuildDelete(data: DiscordPayload) {
     }
   });
 
+  cacheHandlers.forEach("members", (member) => {
+    member
+      .guilds
+      .forEach((_guildMember, guildID) => {
+        if (payload.id === guildID) cacheHandlers.delete("members", member.id);
```

Comment:
What if the member is not only in this guild? Eg the member and the bot share two servers then the bot gets kicked from one the bot shouldn't delete this member entirely from cache.